reV/supply_curve/point_summary.py

```diff
@@ -206,8 +215,29 @@ def mean_lcoe(self):
         mean_lcoe : float | None
             Mean LCOE value for the non-excluded data.
         """
+
         mean_lcoe = None
-        if self.lcoe_data is not None:
+
+        # prioritize the calculation of lcoe explicitly
+        # from the multi year mean CF
+        if self._recalc_lcoe:
+            required = ('fixed_charge_rate', 'capital_cost',
+                        'fixed_operating_cost', 'variable_operating_cost',
+                        'system_capacity')
+            if self.mean_h5_dsets_data is not None:
+                if all(k in self.mean_h5_dsets_data for k in required):
```

Comment:
Do we want to raise an error if the required datasets are not found?

Reply:
There are warnings in the `SupplyCurveAggregation._get_input_data()` method (remember how we talked about warnings/errors in the non-parallel pre-flight checks)